app/models/configuration_script_source.rb

```diff
@@ -0,0 +1,6 @@
+class ConfigurationScriptSource < ApplicationRecord
+  # An Tower project/repo has many playbooks
+  has_many    :configuration_script_payloads, dependent => :destroy
```

Comment:
Ok, I'll remove the `:dependent` and we will let refresh tell us when `playbooks` should be removed.